theLudovyc/GUI/BuildingProducingContainer.gd

```diff
@@ -1,4 +1,5 @@
 extends VBoxContainer
+class_name BuildingProducingContainer
```

Comment:
You do not need to add the class_name specifier if you do not use this class outside